sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -46,7 +46,7 @@ trait DataFromReplacementRule {
  * A version of DataFromReplacementRule that is used when no replacement rule can be found.
  */
 final class NoRuleDataFromReplacementRule extends DataFromReplacementRule {
-  override val operationName: String = "NOT_FOUND"
+  override val operationName: String = "Not supported by GPU: "
```

Comment:
This is redundant, as it later explains on the same line that the operation is not supported.  We don't put "Not supported by GPU: " in front of other lines that will not end up on the GPU, so it's weird to call out this specially.  I think this should just be an empty string.

Reply:
Thank you for your advice. I will change it.